spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -208,6 +208,12 @@ def types
       end
     end
 
+    context 'sensitive' do
+      it 'translates to PSensitiveType' do
+        expect(calculator.infer(PSensitiveType::Sensitive.new("hunter2")).class).to eq(PSensitiveType)
+      end
+    end
```

Comment:
And I'm concerned about trying to limit when we do the type generalization, because we run the risk of missing some code that does `puts o.type.type` and leaks that it's a `String[10, 10]` instead of using some blessed pathway. If we retain sensitive information then we risk leaking it later so wherever possible I think we should destroy any sensitive information as soon as possible.